plugins/outputs/opensearch/sample.conf

```diff
@@ -43,9 +43,17 @@
   # auth_bearer_token = ""
 
   ## Optional TLS Config
```

Comment:
What about including it with `{{template "/plugins/common/tls/client.conf"}}`

Reply:
There was an issue with it but I don't remember what it was... Will take another look...

Reply:
Update: The issue is that the config contains Golang templates itself e.g. in the documentation of the `index_name option`. So the suggested replacement will try to fill in those templates as well and will fail to do so.
```text
2023/11/27 15:23:01 Executing template failed: template: sample.conf.in:13:46: executing "sample.conf.in" at <.Tag>: Tag is not a method but has arguments
```

Reply:
Oh right, these would need to be escaped then in that case.